config/settings.yml

```diff
@@ -895,6 +896,7 @@
     :keep_hourly_performances: 6.months
     :keep_realtime_performances: 4.hours
     :purge_window_size: 1000
+    :queue_timeout: 20.minutes
```

Comment:
do we also need to increase `performance_realtime_purging_interval` to avoid overlapping in scheduler ?